adobe-edge/README.md

```diff
@@ -0,0 +1,71 @@
+# THEOplayer React-Native Adobe Edge Connector
+
+An Adobe analytics connector for `@theoplayer/react-native` using the
```

Comment:
Shouldn't this be `react-native-theoplayer`?

Reply:
Yes true, but we are planning to use a scoped package name in the future, just like the other packages (8.0.0 maybe)?